sdk/netapp/arm-netapp/src/models/index.ts

```diff
@@ -808,176 +924,1262 @@ export interface Snapshot extends BaseResource {
 }
 
 /**
- * revert a volume to the snapshot
+ * Snapshot policy information
  */
-export interface VolumeRevert {
+export interface SnapshotPolicy extends BaseResource {
   /**
-   * Resource id of the snapshot
+   * Resource location
    */
-  snapshotId?: string;
+  location: string;
+  /**
+   * Resource Id
+   * **NOTE: This property will not be serialized. It can only be populated by the server.**
+   */
+  readonly id?: string;
+  /**
+   * Resource name
+   * **NOTE: This property will not be serialized. It can only be populated by the server.**
+   */
+  readonly name?: string;
+  /**
+   * Resource type
+   * **NOTE: This property will not be serialized. It can only be populated by the server.**
+   */
+  readonly type?: string;
+  /**
+   * Resource tags
+   */
+  tags?: { [propertyName: string]: string };
+  /**
+   * hourlySchedule. Schedule for hourly snapshots
+   */
+  hourlySchedule?: any;
+  /**
+   * dailySchedule. Schedule for daily snapshots
+   */
+  dailySchedule?: any;
+  /**
+   * weeklySchedule. Schedule for weekly snapshots
+   */
+  weeklySchedule?: any;
+  /**
+   * monthlySchedule. Schedule for monthly snapshots
+   */
+  monthlySchedule?: any;
+  /**
+   * The property to decide policy is enabled or not
+   */
+  enabled?: boolean;
 }
 
 /**
- * Authorize request
+ * Snapshot policy properties
  */
-export interface AuthorizeRequest {
+export interface SnapshotPolicyDetails {
```

Comment:
There are many such kind of changes, and they are breaking change. So you need bump the major version.